admin-dev/themes/new-theme/template/layout.tpl

```diff
@@ -3,150 +3,131 @@
 <head>
   {$header}
 </head>
+
 <body class="lang-{$iso_user}{if $lang_is_rtl} lang-rtl{/if} {$smarty.get.controller|escape|strtolower}{if $collapse_menu} page-sidebar-closed{/if}">
 
-{* TODO: REPLACE THIS CLASSS SOMEWHERE
-{if $collapse_menu} page-sidebar-closed{/if}
-*}
+{if $display_header}
+  <header id="header">
 
-<header id="header">
-  <nav id="header_infos" class="main-header">
+    <nav id="header_infos" class="main-header">
+      <button class="btn btn-primary-reverse onclick btn-lg unbind ajax-spinner"></button>
 
-    <button class="btn btn-primary-reverse onclick btn-lg unbind ajax-spinner"></button>
+      {* Logo *}
+      <i class="material-icons js-mobile-menu">menu</i>
+      <a id="header_logo" class="logo float-left" href="{$default_tab_link|escape:'html':'UTF-8'}"></a>
+      <span id="shop_version">{$ps_version}</span>
 
-    {* TODO: BUTTON USED FOR THE MOBILE VERSION TO REACTIVATE *}
-    {* TODO: TO REPLACE
-    <button id="header_nav_toggle" type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse-primary">
-      <i class="icon-reorder"></i>
-    </button>
-    *}
+      <div class="component" id="quick-access-container">
+        {include file="components/layout/quick_access.tpl"}
+      </div>
+      <div class="component" id="header-search-container">
+        {include file="components/layout/search_form.tpl"}
+      </div>
 
-    {* Logo *}
-    <i class="material-icons js-mobile-menu">menu</i>
-    <a id="header_logo" class="logo float-left" href="{$default_tab_link|escape:'html':'UTF-8'}"></a>
-    <span id="shop_version">{$ps_version}</span>
+      {if isset($debug_mode) && $debug_mode == true}
+        <div class="component hide-mobile-sm" id="header-debug-mode-container">
+          <a class="link shop-state"
+             id="debug-mode"
+             data-toggle="pstooltip"
+             data-placement="bottom"
+             data-html="true"
+             title="<p class='text-left'><strong>{l s='Your shop is in debug mode.'}</strong></p><p class='text-left'>{l s='All the PHP errors and messages are displayed. When you no longer need it, [1]turn off[/1] this mode.' html=true sprintf=['[1]' => '<strong>', '[/1]' => '</strong>']}</p>"
```

Comment:
Funny we don't use domain in this template :thinking:

Reply:
Translation still works o_O I just checked in French: "Votre boutique est en mode de débogage."

Reply:
Maybe there is some kind of default translation domain being used here